### PR TITLE
update husky config

### PR DESCRIPTION
**Pull request by groninge01**

### CODE REVIEW

File: apps/frontend-v3/package.json
Comment (garethfuller) [inline apps/frontend-v3/package.json:1]: These changes aren't required?

File: package.json
Comment (garethfuller) [inline package.json:1]: Same with these?

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: apps/frontend-v3/package.json
Comment (groninge01) [reply apps/frontend-v3/package.json:1]: No, they are not required. I just like a package.json to be sorted :)
As pnpm doesn't really do this maybe we should add a prettier plugin for this?

Comment (groninge01) [reply apps/frontend-v3/package.json:1]: Sorted the package.json.

File: package.json
Comment (groninge01) [reply package.json:1]: Sorted the package.json.

**Review by garethfuller — APPROVED** [review]

**Pull request merged**